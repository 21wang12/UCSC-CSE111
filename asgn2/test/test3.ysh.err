yshell: cat: /dir/dir3/fle: No such file or directory
