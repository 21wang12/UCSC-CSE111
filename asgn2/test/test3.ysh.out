./yshell build Apr 29 2022 19:04:34
% mkdir dir
% cd dir
% mkdir dir2
% mkdir dir3
% cd dir3
% pwd
/dir/dir3
% make file with stuff
% make another file with stuff ?????? /*$#@!#(){}[]
% cat another
filewithstuff??????/*$#@!#(){}[]
% cat fle 
% echo what happened?  the previous command no such file
what happened? the previous command no such file
% prompt ysh:
ysh: ls .
/dir/dir3/.:
     4       4  ./
     2       4  ../
     6       5  another
     5       2  file
ysh: ls ..
/dir/dir3/..:
     2       4  ./
     1       3  ../
     3       2  dir2/
     4       4  dir3/
ysh: lsr /
/:
     1       3  ./
     1       3  ../
     2       4  dir/
/dir:
     2       4  ./
     1       3  ../
     3       2  dir2/
     4       4  dir3/
/dir/dir2:
     3       2  ./
     2       4  ../
/dir/dir3:
     4       4  ./
     2       4  ../
     6       5  another
     5       2  file
ysh: # Note the error in fle above, it may error out at that
ysh: # point or print another.
ysh: # But the command after the failed cat must be executed.
ysh: # $Id: test3.ysh,v 1.3 2022-04-22 10:41:38-07 - - $
ysh: ^D
yshell: exit(1)
